src/common_main.c

```diff
@@ -67,21 +67,43 @@ static bool _setup_wally(void)
     return wally_set_operations(&_ops) == WALLY_OK;
 }
 
+// Go into bootloader if there was an error during startup, so a firmware update can be
+// applied. Otherwise, if there is an Abort() during startup, there would no way to reboot into the
+// bootloader and the device would be bricked.
+static void _bootloader_autoenter(void)
+{
+    auto_enter_t auto_enter = {
+        .value = sectrue_u8,
+    };
+    upside_down_t upside_down = {
+        .value = screen_is_upside_down(),
+    };
+    if (!memory_bootloader_set_flags(auto_enter, upside_down)) {
```

Comment:
I think the `if` is unnecessary. I would remove it. but the comment of course is good.

Reply:
USE_RESULT prevents me from doing that:

https://github.com/digitalbitbox/bitbox02-firmware/blob/70e31a705ecbad519db3aef539029c44980928df/src/memory/memory.h#L200

And I think that is a good thing. Nice to see that there is a result by the syntax only, not relying on a comment.